docs/platform/webcontainers/browser-config.md

```diff
@@ -22,15 +22,30 @@ With Chrome's defaults, starting with version 118 or later, you should get a pre
 
 Read on if you run into issues or want an even smoother experience.
 
-Previews opened in a separate tab will require you to connect them to the editor. You usually get prompted to click on a button to do that. However, you might do that more frequently than expected with Chrome's [memory saver][CHROME_MEMORY_SAVER]. To avoid it, you can add the following exception in [chrome://settings/performance](chrome://settings/performance):
+### Previews in a separate tab {#chrome-preview-separate-tab}
+
+Previews opened in a separate tab are not connected by default to the editor and require an extra step. You usually get prompted to click a button to do that.
+
+You can have it done automatically by modifying your popups settings, [chrome://settings/content/popups](chrome://settings/content/popups), with the following exception:
```

Comment:
nitpick:

maybe something like: 

This can happen automatically if your popups settings are adjusted with the following extension: